package.json

```diff
@@ -94,11 +94,13 @@
     "boom": "^5.2.0",
     "cids": "^0.5.1",
     "debug": "^3.0.0",
+    "fnv1a": "^1.0.1",
```

Comment:
What is this used for?

Reply:
Sounds good. I actually don't know why is this specific hashing function used, but that is something probably @Kubuxu can fill in?

Reply:
The set serialization structure is very old. I was not here when it was designed, in future we plan migrating to same structure used buy directory sharding or whatever sharded array/map/set the IPLD provides.

From what I know it is same structure that is used by some DB software. FNV1a is generic fast hash function for sharding in this case.